net-im/psi/psi-1.2.ebuild

```diff
@@ -0,0 +1,148 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PLOCALES="be bg ca cs de en eo es et fa fi fr he hu it ja kk mk nl pl pt pt_BR ru sk sl sr@latin sv sw uk ur_PK vi zh_CN zh_TW"
+PLOCALE_BACKUP="en"
+
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.xz"
+
+inherit eutils l10n multilib qmake-utils
+
+DESCRIPTION="Qt4 Jabber client, with Licq-like interface"
+HOMEPAGE="http://psi-im.org/"
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="aspell crypt dbus debug doc enchant +hunspell qt4 +qt5 spell ssl xscreensaver
++plugins whiteboarding webengine webkit"
+
+REQUIRED_USE="
+	spell? ( ^^ ( aspell enchant hunspell ) )
+	aspell? ( spell )
+	enchant? ( spell )
+	hunspell? ( spell )
+	webengine? ( webkit )
+	^^ ( qt4 qt5 )
+"
+
+RDEPEND="
+	net-dns/libidn
+	sys-libs/zlib[minizip]
+	spell? (
+		enchant? ( >=app-text/enchant-1.3.0 )
+		hunspell? ( app-text/hunspell )
+		aspell? ( app-text/aspell )
+	)
+	xscreensaver? ( x11-libs/libXScrnSaver )
+	qt4? (
+		dev-qt/qtgui:4
+		dbus? ( dev-qt/qtdbus:4 )
+		app-crypt/qca:2[qt4]
+		whiteboarding? ( dev-qt/qtsvg:4 )
+		webkit? ( dev-qt/qtwebkit:4 )
+	)
+	qt5? (
+		dev-qt/qtgui:5
+		dev-qt/qtxml:5
```

Comment:
This and other lines: Please keep dependencies sorted, it makes future maintenance much better. Optional dependencies below fixed, and also sorted.